src/software/ai/hl/stp/action/intercept_ball_action.cpp

```diff
@@ -6,7 +6,7 @@
 #include "software/ai/evaluation/robot.h"
 #include "software/ai/intent/move_intent.h"
 #include "software/geom/util.h"
-#include "software/logger/init.h"
+#include "software/logger/Logger.h"
```

Comment:
so uh, minor nit: Filename should be lowercase `logger.h` according to our style guide :sweat_smile:

Reply:
Oh yea I just realized it was capitalized in the ticket. Whoops, sorry about that.

Reply:
:scream: Fixed it.